src/applications/personalization/dashboard-2/selectors.js

```diff
@@ -13,6 +13,5 @@ export const selectShowDashboard2 = state => {
 };
 
 const selectFolders = state => state.health?.msg?.folders;
+export const selectUnreadCount = state => state.health?.msg?.unreadCount;
 export const selectFolder = state => selectFolders(state)?.data?.currentItem;
-export const selectUnreadMessagesCount = state =>
```

Comment:
This selector had only been used by the Health Care component. I'm replacing it with the new selector that uses the new, simplified slice of Redux.